ui/apps/everest/.e2e/release/pitr.e2e.ts

```diff
@@ -446,6 +449,128 @@ test.describe.configure({ retries: 0 });
         }
       });
 
+      test(`PITR restore to new cluster [${db} size ${size}]`, async ({
+        page,
+      }) => {
+        await test.step('Create a backup to restore from', async () => {
+          await gotoDbClusterBackups(page, clusterName);
+          await clickOnDemandBackup(page);
+          await page
+            .getByTestId('text-input-name')
+            .fill(baseBackupName + '-new-cluster');
+          await expect(page.getByTestId('text-input-name')).not.toBeEmpty();
+          await expect(
+            page.getByTestId('text-input-storage-location')
+          ).not.toBeEmpty();
+          await page.getByTestId('form-dialog-create').click();
+          await waitForStatus(
+            page,
+            baseBackupName + '-new-cluster',
+            'Succeeded',
+            240000
+          );
+        });
+
+        await test.step('Add some data after the backup', async () => {
+          await insertTestDB(
+            clusterName,
+            namespace,
+            ['9'],
+            ['1', '2', '3', '4', '5', '6', '7', '8', '9']
+          );
+          // for PG we need one more transaction to be able to restore to the previous one
+          if (db == 'postgresql') {
+            await insertTestDB(
+              clusterName,
+              namespace,
+              ['10'],
+              ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+            );
+          }
+        });
+
+        await test.step('Wait for binlogs to be uploaded', async () => {
+          const delay = (ms: number) =>
```

Comment:
Can you use `await page.waitForTimeout(60000);` here?
I think you should because the parent test includes the `page` object.

Reply:
Done